### PR TITLE
Added the atmosphere licenses

### LINKED ISSUE #1897: Add new atmosphere licenses
See https://www.open-austin.org/atmosphere-license/ We should add them to be able to detect these

### PR DESCRIPTION
**Pull request by Pratikrocks**
Fixes #1897 

Hello all,
In this PR I have added the [atmosphere license](https://www.open-austin.org/atmosphere-license/) 

### Tasks

* [X] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [X] PR is descriptively titled 📑 and links the original issue above 🔗
* [X] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [X] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

File: src/licensedcode/data/licenses/atmosphere.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+key: atmosphere
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/atmosphere.yml:1]: This has a version "Atmosphere Software License Version 0.4" so please use instead `atmosphere-0.4` as a key

File: src/licensedcode/data/licenses/atmosphere.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+short_name: atmosphere
+name: Atmosphere Software License Version 0.4
+category: Copyleft
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/atmosphere.yml:4]: Use `Proprietary Free`

**Comment by Pratikrocks** [comment]
Thanks @pombredanne  for the review ,I have made the changes

**Comment by Pratikrocks** [comment]
@pombredanne , I did the required changes, and rebased to a single commit , the test cases results are also better now ,than the previous (30 successful) commits.

**Comment by pombredanne** [comment]
All good, merging at last! thank you ++

**Pull request merged**